src/data/program_configuration.js

```diff
@@ -386,7 +394,7 @@ export default class ProgramConfiguration {
             const useIntegerZoom = value.property.useIntegerZoom;
 
             if (value.value.kind === 'constant') {
-                self.binders[property] = new ConstantBinder(value.value, name, type);
+                self.binders[property] = new ConstantBinder(value.value.value, name, type);
```

Comment:
This one's weird — I guess this was a buggy code in an unused code path, since this doesn't break any tests.

Reply:
And flow isn't helping us here because `ConstantBinder` is deducing the type from that first argument? 😕

Reply:
Yeah, I guess it's because of flow templating (`ConstantBinder<T>`) — I'm not fully sure how it works (@jfirebaugh may know better) but looks like if there's a template or an `any` in Flow, you can basically do anything in the code the touches those places without any type safety.